pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: layout.pug
 navigationTitle: Install Kaptain on DKP 2.x
-title: Install Kaptain on DKP 2.x
+title: Install Kaptain on DKP 2.x (non air-gapped)
```

Comment:
do we want to call that out that explicitly? I'd vote for calling out the air gapped part rather than the non air gapped part

Reply:
In other places in the docs we refer to this as a networked environment:

https://docs.d2iq.com/dkp/kommander/2.2/install/networked/

```suggestion
title: Install Kaptain on DKP 2.x in a networked environment
```

Reply:
Yep, you are completely right! Thanks, Craig!